src/test/java/com/github/sardine/DavFolderTest.java

```diff
@@ -0,0 +1,32 @@
+package com.github.sardine;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Properties;
+
+/**
+ * Created by Deinok on 08/03/2016.
+ */
+public class DavFolderTest {
+
+    private static final String TEST_PROPERTIES_FILENAME = "test.properties";
+    private Properties properties;
+    private Sardine sardine;
+
+    @Before
+    public void before() throws IOException {
+        this.properties = new Properties();
+        this.sardine = SardineFactory.begin("jgg15", "warhammer2611");
+    }
+
+    @Test
+    public void redirectTest1() throws IOException, URISyntaxException {
+        DavFolder davFolder = this.sardine.getRoot("https://cv.udl.cat/dav/102006-1516/");
```

Comment:
I'd rather not have tests hitting a remote server. We are moving away from this.